docs/userguide/objective.md

```diff
@@ -1,67 +1,92 @@
-targets (if applicable).
-
-To create an objective, it is necessary to provide the following:
-* An optimization `mode`: This can be either `SINGLE` or `DESIRABILITY`,
-  denoting the optimization of a single target function or a combination of
-  different target functions respectively.
-* A list of `targets`: The list of targets that are optimized (see user guide for
-  [`Target`](../../userguide/targets)). Note that the `SINGLE` mode also requires a
-  list containing the single target.
+Optimization problems involve either a single target quantity of interest or 
+several (potentially conflicting) targets that need to be considered simultaneously.  
+BayBE uses the concept of an [`Objective`](baybe.objective.Objective) to allow the user
+to control how these different types of scenarios are handled.
 
 ```{note}
-We are actively working on adding more objective modes for multiple targets.
+We are actively working on adding more objective types for multiple targets.
 ```
 
-## Supported Optimization Modes
-Currently, BayBE offers two optimization modes.
-
-### SINGLE
-In `SINGLE` mode, objectives focus on optimizing a single target. 
-Nearly [examples](../../examples/examples) use this objective mode.
-
+## SingleTargetObjective
+The need to optimize a single [`Target`](baybe.targets.base.Target) is the most basic
+type of situation one can encounter in experimental design. 
+In this scenario, the fact that only one target shall be considered in the design is
+communicated to BayBE by wrapping the target into a
+[`SingleTargetObjective`](baybe.objectives.single.SingleTargetObjective):
 ```python
 from baybe.targets import NumericalTarget
 from baybe.objectives import SingleTargetObjective
 
-target_1 = NumericalTarget(name="yield", mode="MIN", bounds=(0, 100))
-objective = SingleTargetObjective(target=target_1)
+target = NumericalTarget(name="Yield", mode="MAX")
+objective = SingleTargetObjective(target)
 ```
+In fact, the role of the
+`SingleTargetObjective` is to merely signal the absence of other `Target`s in the
+optimization problem.
+Because this fairly trivial conversion step requires no additional user configuration,
+we provide a convenience constructor for it:
 
-### DESIRABILITY
-The `DESIRABILITY` mode enables the combination multiple targets via scalarization 
-into a single value, a method also utilized in classical DOE.
+````{admonition} Convenience construction and implicit conversion
+:class: tip
+* The conversion from a single [`Target`](baybe.targets.base.Target) to a
+[`SingleTargetObjective`](baybe.objectives.single.SingleTargetObjective) describes a
+one-to-one relationship and can be triggered directly from the corresponding target
+object:
+  ```python
+  objective = target.to_objective()
+  ```
+* Also, other class constructors that expect an 
+[`Objective`](baybe.objectives.base.Objective)
+object (such as [`Campaigns`](baybe.campaign.Campaign)) will happily swallow
```

Comment:
I'd prefer "accept" instead of "swallow" (but can live with both).

Reply:
changed 👍🏼